about/people/staff.md

```diff
@@ -2,13 +2,15 @@
 
 - Amanda Bartley, arXiv Administrator
 - Shamsi Brinn, arXiv User Experience Specialist
+- Christopher Cameron, Computational Social Scientist
 - Brian Caruso, Senior arXiv Developer
 - Jim Entwood, Head of Content
 - David Fielding, Senior arXiv Developer
 - Alison Fromme, Community Engagement and Membership Coordinator
 - Martin Lessmeister, Head of Technology
 - Brian Maltzan, Backend Python Developer
 - Rebecca Rich Goldweber, Senior Production Editor
+- Donna Rose, Administrative Assistant
```

Comment:
Could we remove Donna until she confirms she wants to be on our page?

Reply:
yes, sorry about that